internal/core/src/storage/loon_ffi/ffi_reader_c.cpp

```diff
@@ -78,22 +101,28 @@ NewPackedFFIReaderWithManifest(const char* manifest_content,
                                struct ArrowSchema* schema,
                                char** needed_columns,
                                int64_t needed_columns_size,
-                               CFFIPackedReader* c_packed_reader,
+                               CFFIPackedReader* c_loon_reader,
                                CStorageConfig c_storage_config,
                                CPluginContext* c_plugin_context) {
```

Comment:
`c_plugin_context` unused in `NewPackedFFIReaderWithManifest`/`NewPackedFFIReader`?